packages/ds/src/admin/ui/Tooltip.tsx

```diff
@@ -18,10 +19,17 @@ export default function Tooltip({
   children,
   className,
   tooltipClassName,
+  isModal = false,
```

Comment:
```suggestion
  forceRefresh = false,
```
dans le DS on ne doit pas connaitre le contexte extérieur, peux importe que ça soit une modal ou autre chose. la question est qu'elle comportement est attendu du composant en mode boite noir.